Sources/Controllers/Resources/OARequiredMapsResourceViewController/OARequiredMapsResourceViewController.mm

```diff
@@ -277,7 +290,16 @@ - (UITableViewCell *)getRow:(NSIndexPath *)indexPath
             [cell.button setTitleColor:[UIColor colorNamed:ACColorNameTextColorActive] forState:UIControlStateHighlighted];
             cell.button.tag = indexPath.section << 10 | indexPath.row;
             [cell.button removeTarget:self action:NULL forControlEvents:UIControlEventTouchUpInside];
-            [cell.button addTarget:self action:@selector(onCalculateOnlineButtonPressed:) forControlEvents:UIControlEventTouchUpInside];
+
+            if ([item.title isEqualToString:OALocalizedString(@"calculate_online")])
```

Comment:
It is bad practice to use localized title to distinguish items. Use `key` property instead.

Reply:
fixed